pkg/frontend/mysql_protocol.go

```diff
@@ -2359,15 +2359,6 @@ func (mp *MysqlProtocolImpl) appendResultSetBinaryRow(mrs *MysqlResultSet, rowId
 					return err
 				}
 			}
-			// case defines.MYSQL_TYPE_TIMESTAMP:
```

Comment:
You sure that we do not need to handle TIMESTAMP here?

Reply:
TIMESTAMP has been processed in the same case as DATETIME before refactoring

Reply:
![image](https://github.com/user-attachments/assets/fbbf8e3f-08a2-4fd2-b9ea-11d3cee6f9df)
The encoding methods of DATETIME and TIMESTAMP are the same in binary protocol